kvmagent/kvmagent/plugins/vm_plugin.py

```
@@ -4796,23 +4796,52 @@ def hotplug_cpu(self, cpu_num):
+            try:
+                sysinfo = self.domain_xmlobject.get_child_node('sysinfo')
+                if sysinfo is not None:
+                    system = sysinfo.get_child_node('system')
+                    if system is not None:
+                        for entry in system.get_child_node_as_list('entry'):
+                            if entry.name_ == 'manufacturer' and entry.text_ == _SMBIOS_MANUFACTURER:
+                                logger.debug('SMBIOS already set for vm[uuid:%s], '
+                                             'skip QGA cpu online' % self.uuid)
+                                return
+            except Exception:
+                pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)